protobuf/schema/v1/schema.proto

```diff
@@ -1,3 +1,10 @@
+/**
```

Comment:
PR check failed. This is due to comment format. Java-style comments are not compatible - https://protobuf.dev/programming-guides/proto3/#adding-comments